src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
When we add a config value, the person adding the config value gets to decide that policy through config value option (e.g. CLRConfig::IgnoreHKLM) on a per config option basis.  Right now there is no single policy governing for all jit config values - see clrconfigvalues.h

Jit-EE call  getIntConfigValue()/getStringConfigValue() in turn calling CLRConfig::GetConfigValues() with a config option of CLRConfig::REGUTIL_default, which means ignore app config file while reading config value.

The fact now RyuJIT going to read all config values using this API will introduce back compat issue on desktop.  For e.g. complus_UseLegacyJIt is a setting we allow even to come from an app config file.  Post this change RyuJIt will no longer honor this setting in app config files.  There are numerous such clrconfig values.

This definitely needs to be addressed before we check-in.